docs/kernels.rst

```diff
@@ -8,7 +8,7 @@ A 'kernel' is a program that runs and introspects the user's code. IPython
 includes a kernel for Python code, and people have written kernels for
 `several other languages <https://github.com/jupyter/jupyter/wiki/Jupyter-kernels>`_.
 
-When Jupyter starts a kernel, it passes it a connection file. This specifies
+At startup, Jupyter passes the kernel a connection file. This specifies
```

Comment:
I would state `At kernel startup` to disambiguate between Jupyter startup and kernel startup for readers.

Reply:
Thanks for the 👀 @captainsafia. I've made the suggested change